### PR TITLE
[FEATURE] remove a single attribute acceptance tests

### PR DESCRIPTION
**Pull request by iariasleon**
@fgalan

### CODE REVIEW

**Comment by jcanonav** [comment]
LGTM

File: test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+      | metadatas_name   | very_hot    |
+      | metadatas_type   | alarm       |
+      | metadatas_value  | hot         |
+    And verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    When delete an attribute "temperature_0" in entities with id "room"
+      | parameter       | value                                  |
+      | entities_number | the same value of the previous request |
+    Then verify that receive several "No Content" http code
+    And verify that the attribute is deleted into mongo
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature]: If the test case is about removing the attribute in the 3 entities in a single operation, this maybe should be more explicit, e.g.:

"And verify that the attribute is deleted into mongo in the "3" entities"

or something similar

File: test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+  @service_path_delete_error
+  Scenario Outline: Try to delete an attribute by entity ID using NGSI v2 with wrong service path header values
+    Given a definition of headers
+      | parameter          | value                          |
+      | Fiware-Service     | test_delete_service_path_error |
+      | Fiware-ServicePath | <service_path>                 |
+      | Content-Type       | application/json               |
+    When delete an attribute "temperature_0" in entities with id "room"
+    Then verify that receive several "Bad Request" http code
+    And verify several error responses
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature]: You say "several", but actually the response is only one (composed of error and description subfields)

File: test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature]: Fixed in 4b86d0b

File: test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature]: Fixed in 4b86d0b

**Comment by fgalan** [comment]
LGTM

**Pull request merged**